trio/_tests/test_windows_pipes.py

```diff
@@ -1,37 +1,47 @@
+from __future__ import annotations
+
 import sys
-from typing import Any, Tuple
+from typing import TYPE_CHECKING, Any
 
 import pytest
 
 from .. import _core
 from ..testing import check_one_way_stream, wait_all_tasks_blocked
 
+# Mark all the tests in this file as being windows-only
+pytestmark = pytest.mark.skipif(sys.platform != "win32", reason="windows only")
+
+assert (
+    sys.platform == "win32" or not TYPE_CHECKING
+)  # Skip type checking when not on Windows
+
 if sys.platform == "win32":
     from asyncio.windows_utils import pipe
 
     from .._core._windows_cffi import _handle, kernel32
     from .._windows_pipes import PipeReceiveStream, PipeSendStream
 else:
-    pytestmark = pytest.mark.skip(reason="windows only")
-    pipe: Any = None
-    PipeSendStream: Any = None
-    PipeReceiveStream: Any = None
+    pipe = Any
+    _handle = Any
+    kernel32 = Any
+    PipeReceiveStream = Any
+    PipeSendStream = Any
```

Comment:
Types here shouldn't matter. In fact, we probably can delete these?

Reply:
I wanted to delete them as well, but Mypy has failures otherwise. e74adb211d9733128c5bd337e890c9b26931356f added this to fix the errors in this run: https://github.com/python-trio/trio/actions/runs/6445455847/job/17499532948

Reply:
Huh, that's strange. Maybe now it will work after https://github.com/python-trio/trio/pull/2812/commits/ee64fe3a68f25f6ba62827d2dd3d5f7087f14d2f

Reply:
Seems to work now after removing in 5c4529c, good catch!